rocAL/rocAL/include/pipeline/tensor.h

```diff
@@ -164,6 +165,10 @@ class rocalTensorInfo {
         _batch_size *= sequence_length;
     }
     size_t get_channels() const { return _channels; }
+    void copy_roi(void *roi_buffer) {
```

Comment:
Are we copying the roi region or roi values?
if it is roi values, can we rename it to something to copy_roi_dims() or something like that?

Reply:
its roi - xywh